There are two tests that are failing right now on Snuba test suite:

```
Traceback (most recent call last):
  File "/Users/ted/workspaces/sentry/sentry/tests/snuba/tagstore/test_tagstore_backend.py", line 149, in test_get_group_tag_key
    assert set(keys) == set(['baz', 'environment', 'foo', 'sentry:release'])
AssertionError: assert set(['baz', '...', 'release']) == set(['baz', 'e...try:release'])
  Extra items in the left set:
  'release'
  Extra items in the right set:
  'sentry:release'
  Use -v to get the full diff
```

This is expected right now — I think this is internal to the Snuba service and needs to be fixed there?

```
Traceback (most recent call last):
  File "/Users/ted/workspaces/sentry/sentry/tests/snuba/tagstore/test_tagstore_backend.py", line 101, in test_get_group_tag_keys_and_top_values
    assert result[3]['topValues'][0]['value'] == '100'
AssertionError: assert '200' == '100'
  - 200
  + 100
```

This doesn't actually have a stable sort beyond times seen here (right now this is a tie between two items that have 1 value.) I wonder if we should include the value as a secondary sort parameter/tiebreaker?

--- a/tests/snuba/tagstore/test_tagstore_backend.py
+++ b/tests/snuba/tagstore/test_tagstore_backend.py
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 
 import calendar
-from datetime import datetime, timedelta
+from datetime import timedelta
 import json
 import pytest
 import requests
@@ -270,7 +270,6 @@ class TagStorage(TestCase):
             EventUser(project_id=self.proj1.id, email='user1@sentry.io'),
             EventUser(project_id=self.proj1.id, ident='user2')
         ])
-        two_seconds_ago = self.now - timedelta(seconds=2)
         assert len(result) == 2
         result.sort(key=lambda x: x.value)
         assert result[0].value == 'user1'

--- a/tests/snuba/tagstore/test_tagstore_backend.py
+++ b/tests/snuba/tagstore/test_tagstore_backend.py
@@ -81,6 +81,7 @@ class TagStorage(TestCase):
         assert requests.post(settings.SENTRY_SNUBA + '/tests/insert', data=data).status_code == 200
 
     def test_get_group_tag_keys_and_top_values(self):
+        # TODO: `release` should be `sentry:release`
         result = self.ts.get_group_tag_keys_and_top_values(
             self.proj1.id,
             self.proj1group1.id,
@@ -98,7 +99,10 @@ class TagStorage(TestCase):
         assert result[3]['key'] == 'release'
         assert result[3]['uniqueValues'] == 2
         assert result[3]['totalValues'] == 2
-        assert result[3]['topValues'][0]['value'] == '100'
+        top_release_values = result[3]['topValues']
+        assert len(top_release_values) == 2
+        assert set(v['value'] for v in top_release_values) == set(['100', '200'])
+        assert all(v['count'] == 1 for v in top_release_values)
 
     def test_get_top_group_tag_values(self):
         resp = self.ts.get_top_group_tag_values(

